test/tstInterpDetailsCRBF.cpp

```diff
@@ -0,0 +1,97 @@
+/****************************************************************************
+ * Copyright (c) 2023 by the ArborX authors                                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "ArborX_EnableDeviceTypes.hpp"
+#include "ArborX_EnableViewComparison.hpp"
+#include <interpolation/details/ArborX_InterpDetailsCompactRadialBasisFunction.hpp>
+
+#include "BoostTest_CUDA_clang_workarounds.hpp"
+#include <boost/math/tools/polynomial.hpp>
```

Comment:
This introduces a new dependency on `boost::math`, I think. According to [this](https://www.boost.org/doc/libs/1_83_0/more/getting_started/unix-variants.html#header-only-libraries), it only has optional compiled binary, but can be used header only. So I think we don't need to update CMake's boost `REQUIRED_COMPONENTS`.